tests/e2e_tests/cypress/support/generate_stress_test_data.py

```diff
@@ -0,0 +1,107 @@
+import json
+import random
+import uuid
+
+
+def generate_random_position_within_poland():
```

Comment:
Ok. I had to think that through... although I like randomness approach it probably is not best to have it in stress test. Let's keep same dataset for every run to be sure we have the same condition.

Reply:
Set seed. Now points will always generate in the same way